pip-freeze.txt

```diff
@@ -27,7 +27,7 @@ django-redis==4.8.0
 django-rosetta==0.7.13
 django-storages==1.5.2
 django-timezone-field==2.0  # via rapidpro-dash
-django==1.11.7
+django==1.11.9
```

Comment:
to get change by @dodobas that makes Django user server-side cursors with values_list (needed for next part)